src/org/thoughtcrime/securesms/util/task/ProgressDialogAsyncTask.java

```diff
@@ -12,6 +16,8 @@
   private       ProgressDialog         progress;
   private final String                 title;
   private final String                 message;
+  private       boolean                cancellable;
```

Comment:
"cancellable" (two "l") is british. And "cancelable" is used in other places already.

Reply:
thanks, i'll fix that